### PR TITLE
Fix displaying error messages on unsupported layers

### PR DESCRIPTION
**Pull request by csillag**
Before:

image

After:

image

### CODE REVIEW

**Comment by buberdds** [comment]
when we render layout with unsupported layer, do we need to hide network/ParaTime picker?

**Comment by csillag** [comment]
> when we render layout with unsupported layer, do we need to hide network/ParaTime picker?

In the current for, yes we should, because otherwise it might give us further errors.
However, showing it could be the easiest way back to a valid URL.... but in order to be able to show it, we need to teach it to handle launching with an invalid (or disabled) layer. I don't want to add code changes to it now, since there is another pending PR. 

So for now I will just hide it. User can still click on the Oasis logo and start overf.

**Comment by csillag** [comment]
Updated the code accordingly.

File: src/app/hooks/useScopeParam.ts [hunk 1]
```diff
@@ -10,38 +10,50 @@ export const useNetworkParam = (): Network | undefined => {
...
+    valid: true,
   }
 
-  if (!scope || !RouteUtils.getEnabledNetworks().includes(scope.network))
-    throw new AppError(AppErrors.UnsupportedNetwork)
+  if (network === undefined || layer === undefined) {
+    scope.valid = false
+    if (!error)
+      throw new Error(
+        'You must either specify both network and layer or none of them. You can not have one but not the other.',
```
Comment (lukaw3d) [inline src/app/hooks/useScopeParam.ts:36]: Why are we encoding this here? Isn't it a consequence of routes, not scope param, so add a 404 route instead

**Pull request merged**